core/app/c/[communitySlug]/pubs/[pubId]/page.tsx

```diff
@@ -128,7 +148,7 @@ export default async function Page({
 							);
 						})}
 				</div>
-				<div className="flex w-64 flex-col gap-4 rounded-lg bg-gray-50 p-4 font-semibold shadow-inner">
+				<div className="flex w-96 flex-col gap-4 rounded-lg bg-gray-50 p-4 font-semibold shadow-inner">
```

Comment:
I made this wider to fit the member management interface. It's about the same width as the stage panel now